verification/curator-service/api/test/sources.test.ts

```diff
@@ -29,8 +33,31 @@ afterAll(() => {
     return mongoose.disconnect();
 });
 
-beforeEach(() => {
-    return Source.deleteMany({});
+beforeEach(async () => {
+    await Source.deleteMany({});
+    await User.deleteMany({});
+    await Session.deleteMany({});
+});
+
+afterAll(async () => {
+    await Source.deleteMany({});
+    await User.deleteMany({});
+    await Session.deleteMany({});
+});
+
+let curatorRequest: any;
+beforeEach(async () => {
```

Comment:
Is there any reason to have two top-level beforeEach and afterAll stanzas? Did you mean to nest this, or something else?

Reply:
This is because I wanted a user for each test, I could in theory reuse the same agent but as superagent stores cookies/other state I felt it would be cleaner to have a user per test.